### DISCUSSION TITLE
Unable to load SDRplay API library from default install path

### DISCUSSION THREAD

**Discussion by RevBaker** [DSheirer/sdrtrunk#1809] [Q&A] [answered]
The app log shows the following after an upgrade install to 6.0 final beta:

```
20240126 111637.025 [main] INFO  i.g.d.s.t.sdrplay.api.SDRplay - Loading API Library from default install path: /usr/local/lib/libsdrplay_api.so  [15MB/28MB 55%]
20240126 111637.025 [main] ERROR i.g.d.s.t.sdrplay.api.SDRplay - Unable to load SDRplay API library from default install path.  Loading from java system library path  [15MB/28MB 55%]
20240126 111637.026 [main] WARN  i.g.d.s.t.sdrplay.api.SDRplay - SDRPlay API library not found/installed on this system.  Ensure the API is installed either in the default install location or the install location is included in the 'java.library.path' JVM property contains path to the library file [libsdrplay_api.so].  Current library path property contents: /usr/local/lib  [15MB/28MB 55%]
20240126 111637.026 [main] INFO  i.g.d.s.t.sdrplay.api.SDRplay - API library was not loaded  [15MB/28MB 55%]
20240126 111637.026 [main] INFO  i.g.d.s.t.sdrplay.api.SDRplay - API library is not available.  [15MB/28MB 55%]
```

This library is not on the drive and was not installed elsewhere. ldconfig -p | grep sdrplay  does not find it in the /etc/ld.so.cache.
I've seen references to - libsdrplay_api.so.3.07 => /usr/local/lib/libsdrplay_api.so.3.07 - in other forums but cannot find that either:

```
SoapySDRUtil --probe="driver=libsdrplay_api.so.3.07"
######################################################
##     Soapy SDR -- the SDR abstraction library     ##
######################################################

Probe device driver=libsdrplay_api.so.3.07
Error probing device: SoapySDR::Device::make() no match
```

Perplexed. This all stems from an investigation of a failure of the audio which will play on startup and then one of the audio channels will freeze and the other will stop a short time after. JMBE seems to be working as it should. This is only happening with a trunked APCO 25 system that was flawless under v0.5.0-alpha6. Any hints on what I'm missing?

**Comment by DSheirer** [comment]
Are you using an SDRPlay tuner?  If not, then these log messages are inconsequential ... you can ignore them.

**Reply by RevBaker** [reply, marked as answer]
Copy all....Thanks!

But this audio issue is interesting. If I select the DEFAULT STEREO or DEFAULT MONO the audio plays for a bit then quits...the test buttons always work, tho.  I'll ferret this out yet.....

Cheers.

**Comment by DSheirer** [comment]
Closing.
